psutil/__init__.py

```diff
@@ -1545,8 +1544,8 @@ def cpu_times_percent(interval=None, percpu=False):
     def calculate(t1, t2):
         nums = []
         all_delta = sum(t2) - sum(t1)
-        for field in t1._fields:
-            field_delta = getattr(t2, field) - getattr(t1, field)
+        for field in t2._fields:
+            field_delta = getattr(t2, field) - getattr(t1, field, 0)
```

Comment:
Perhaps it's better to think of this as an `izip_longest` filling the shorter CPU values tuple with 0s in missing slots? 

In a any event, I've updated the PR to revert this change – I'll hopefully be able to take a look at your other comments within a few hours.

Thanks for the help & feedback! :)